net-libs/openmq-cclient/openmq-cclient-4.4.1.ebuild

```diff
@@ -73,6 +73,12 @@ src_prepare() {
 	eautoreconf
 }
 
+src_configure() {
+	econf --disable-static
+}
+
 src_install() {
-	emake install DESTDIR="${D}"
+	default
+	dodoc "${WORKDIR}"/mq/src/doc/en/*
```

Comment:
`dodoc -r "${WORKDIR}"/mq/src/doc/en/.`
always try to avoid globs if possible

Reply:
Fixed, thanks